doc/sphinx/dune

```diff
@@ -5,6 +5,6 @@
  (deps (source_tree ../tools/coqrst) README.template.rst)
  (action (run ../tools/coqrst/regen_readme.py %{targets})))
 
-(rule
- (alias refman-html)
- (action (diff README.rst README.gen.rst)))
+; (rule
+;  (alias refman-html)
+;  (action (diff README.rst README.gen.rst)))
```

Comment:
Was this commented out because of [#Rocq devs &amp; plugin devs > doc/sphinx/README fails @ 💬](https://rocq-prover.zulipchat.com/#narrow/channel/237656-Rocq-devs-.26-plugin-devs/topic/doc.2Fsphinx.2FREADME.20fails/near/509684375)?

Did you intend to leave it commented out in the merged code?

Reply:
I did not intend

Reply:
https://github.com/rocq-prover/rocq/pull/20706